Adyen/adyen-react-native
# Feature: Create Expo managed workflow plugin

## Summary
Today `@adyen/react-native` **is not supported by Expo**.

This pull request aims to create an Expo configuration plugin for the library, enabling the community to incorporate it into Expo-managed workflow.

## Tested scenarios

1. Run `expo prebuild --clean` in the `example-expo` app
2. Ensure the **[native configuration](https://github.com/Adyen/adyen-react-native?tab=readme-ov-file#installation)** has been well implemented by looking at:
- `./example-expo/ios/ExampleExpo/AppDelegate.mm`
- `./example-expo/android/app/src/main/java/com/onebasketdemo/expo/MainActivity.java`
- `./example-expo/android/app/src/main/AndroidManifest.xml`
5. Local build through XCode & Android Studio
6. Development and development of client builds through EAS


**Fixed issue**:  #108
